webrender/src/render_backend.rs

```diff
@@ -416,6 +427,10 @@ impl RenderBackend {
         self.frame.discard_frame_state_for_pipeline(pipeline_id);
     }
 
+    fn device_pixel_ratio(&self) -> f32 {
```

Comment:
You might consider renaming this to something like 'accumulated_scale_factor' or something like that. In my mind device_pixel_ratio is pegged to be the HiDPI scaling factor, so I think this could be potentially confusing to others as well.